optuna/structs.py

```diff
@@ -18,6 +18,18 @@
 
 _logger = logging.get_logger(__name__)
 
+message = (
```

Comment:
Let's make this private. Or better, inline the string in the `warning.warn` call.
```suggestion
_message = (
```